### ISSUE TITLE
Public publish to NPM registry

### ISSUE THREAD

**Issue by boian-ivanov** [webgme/svelte-jsonschema-form#25]
Hello, I found this package as I was searching for exactly this functionality, but it doesn't seem to be publicly installable via the NPM registry. Is that something that you're looking to do at some point, or is it only for internal use?

**Comment by brollb** [comment]
Yeah, we plan on publishing to npm. Our current focus has been developing this for use in https://github.com/webgme/webgme-taxonomy/ as part of a research project at Vanderbilt University.

Contributions are welcome (though we currently don't have a lot of cycles to prioritize things outside of our current roadmap, unfortunately).

**Comment by suenot** [comment]
need publish to npm
